README.md

```diff
@@ -182,7 +182,7 @@ and VMware Cloud on AWS APIs.
 
 * Linux/Mac:
 
-    export PYTHONPATH=${PWD}:$PYTHONPATH
+    export PYTHONPATH=\$\{PWD\}:\$PYTHONPATH
```

Comment:
Do we need to escape?

Reply:
reverted this change

Reply:
You can preview the changes for .md files